snippets/matter-power-consumption-tests/boards/nrf54l15dk_nrf54l15_cpuapp.conf

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Disable LEDs that lead to the current leakages
+CONFIG_NCS_SAMPLE_MATTER_LEDS=n
+
+# Disable watchdog that increases the sleep current
+CONFIG_NCS_SAMPLE_MATTER_WATCHDOG=n
+
+# Increase MPSL calibration period to prevent too often CPU wake-ups.
+CONFIG_MPSL_CALIBRATION_PERIOD=60000
```

Comment:
why does this need a specific kconfig fragment for the board, why can't it go in the normal snippet .conf?

Reply:
Because, the nRF54L15 needs few additional tweaks to lower power consumption. On other platforms this is not required (this particular Kconfig option is even not used).

Reply:
so disabling it isn't a problem then, that way there is a snippet that works for all boards including customer boards, whereas right now this works for no nrf54l* customer boards

Reply:
This is not targeted for the customers use, and we also do not mention how to use it in the documentation. This is added for the QA to avoid making handling multiple configurations in their Test Automation System.

This works on all boards, but we don't want to use it with all boards, because this is not required. Only nRF54L15 DK (not SoC) has problems with LEDs current leakage, so I don't see a reason to disable it for other/customer boards as well. Similarly, we configure nrf dks in samples' boards directory and this does not apply to the customers' boards, so I think we are not doing anything different here.

Reply:
Just to be clear, as maybe I haven't put it explicitly. Disabling this modules is not something we would like to do, ideally we would like to keep it enabled. Unfortunately, the nRF54L15 DK requires this sort of "workaround", so we don't want to put workaround, where it is not required.

Reply:
Here is the justification: https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/protocols/matter/getting_started/low_power_configuration.html#disable_leds_module